templates/buyer/product/show.html.twig

```diff
@@ -21,7 +21,14 @@
             <div class="col-lg-7 col-md-6">
                 <div class="product-details-view-content sp-normal-content pt-60">
                     <div class="product-info">
-                        <h2>{{ product.name }}</h2>
+                        <h2>
+                            {{ product.name }}
+                            {% if product.quantity > 0 %}
+                                <span class="item-quantity" style="font-size: 15px;color:green;"><b>({{ product.quantity }}</b> items in stock)</span>
```

Comment:
inline styles!!

please move them to css files

Reply:
done